examples/tracing/tracing.server.out

```diff
@@ -0,0 +1,9 @@
+# To start the service, navigate to the directory that contains the
```

Comment:
Since the example reports tracers to Jaeger, I think it would be useful to tell the user how to setup Jaeger and view the tracers. WDYT?